### DISCUSSION TITLE
Need help with crash in iOS

### DISCUSSION THREAD

**Discussion by baubrey91** [maplibre/maplibre-native#2632] [Q&A]
New Relic is reporting a bunch of crashes I am unable to reproduce. I imported the dSYM and have the stack trace. It is a sigsev crash. Would anyone have any idea what the issue is that is causing this? ![image](https://github.com/user-attachments/assets/c44753e0-1a52-4b63-a1b5-2ea77bb1c436)

**Comment by louwers** [comment]
Could you create an issue? Make sure to put a descriptive title and fill in the details of the bug report template. Thanks.
